### PR TITLE
[chore]: create base class for accordion item and remove style and layout specific api

### PR DESCRIPTION
**Pull request by eljefe223**
## Previous Behavior

Style and layout API was bundled into one `AccordionItem` class.

## New Behavior

Abstracts out style and layout API and creates `AccordionItem` class.

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/accordion-item/accordion-item.ts [hunk 1]
```diff
@@ -101,46 +101,59 @@ export class AccordionItem extends FASTElement {
...
    *
    * @public
    * @remarks
-   * HTML Attribute: size
+   * HTML Attribute: block
    */
-  @attr
-  public size?: AccordionItemSize;
+  @attr({ mode: 'boolean' })
+  public block: boolean = false;
```
Comment (davatron5000) [inline packages/web-components/src/accordion-item/accordion-item.ts:111]: This may be presentational per some mockups of accordions I've seen recently.

**Review by davatron5000 — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: packages/web-components/src/accordion-item/accordion-item.ts (hunk 1 above)
Comment (eljefe223) [reply packages/web-components/src/accordion-item/accordion-item.ts:111]: yup, I'll move.

**Pull request merged**